internal/cli/atlas/integrations/create/new_relic.go

```diff
@@ -58,13 +59,15 @@ func (opts *NewRelicOpts) Run() error {
 	return opts.Print(r)
 }
 
-func (opts *NewRelicOpts) newNewRelicIntegration() *atlas.ThirdPartyIntegration {
-	return &atlas.ThirdPartyIntegration{
-		Type:       newRelicIntegrationType,
-		LicenseKey: opts.licenseKey,
-		AccountID:  opts.accountID,
-		WriteToken: opts.writeToken,
-		ReadToken:  opts.readToken,
+func (opts *NewRelicOpts) newNewRelicIntegration() *atlasv2.Integration {
+	return &atlasv2.Integration{
+		NewRelic: &atlasv2.NewRelic{
+			Type:       pointer.Get(newRelicIntegrationType),
```

Comment:
```suggestion
			Type:       &newRelicIntegrationType,
```

Reply:
`&` doesn't work here as newRelicIntegrationType is const.

Reply:
feel free to change it to be a `var`